src/web/src/scan_run.ecpp

```diff
@@ -186,27 +188,24 @@ UserInfo user;
 
 
     if (rv == -1)
-        http_die ("internal-error", "mlmpool.get () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("mlmpool.get () failed."));
     else
     if (rv == -2)
-        http_die ("internal-error", "client->sendto () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("client->sendto () failed."));
     else
     if (rv == -3)
-        http_die ("internal-error", "client->recv () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("client->recv () failed."));
     else
     if (rv == -4)
-        http_die ("request-param-required", "<range>");
+        http_die ("internal-error", TRANSLATE_ME ("fty-discovery returned error."));
     else
     if (rv == -5)
-        http_die ("internal-error", "fty-discovery returned malformed or unexpected message.");
+        http_die ("internal-error", TRANSLATE_ME ("fty-discovery returned malformed or unexpected message."));
     else
     if (rv == -6)
-        http_die ("internal-error", "fty-discovery precedent scan currently stopping.");
+        http_die ("internal-error", TRANSLATE_ME ("fty-discovery precedent scan currently stopping."));
     else
     if (rv == -7)
-        http_die ("internal-error", "fty-discovery already have a scan running.");
-    else
-    if (rv == -8)
-        http_die("bad-request-document", "Please check RFC-11 for valid json schema description.");
```

Comment:
Sanity check: should this message (and return code handling) have disappeared? Did the function change and/or was it wrong before?

Reply:
This is supposed to reflect the change described above.